### PR TITLE
feat(components/indicators): remove bottom margin from alert component

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/648/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/popovers-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/648/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/648/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/648/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/648/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/648/playground/)

File: libs/components/packages/src/schematics/migrations/migration-collection.json [hunk 1]
```diff
@@ -14,6 +14,11 @@
       "version": "",
       "factory": "./update-7/rename-mutation-observer-service",
       "description": "Replace all instances of 'MutationObserverService' with 'SkyMutationObserverService'"
+    },
+    "add-compat-stylesheets": {
+      "version": "",
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/src/schematics/migrations/migration-collection.json]: Looks like I mistakenly forgot to put a version in the `rename-mutation-observer-service` schematic. Mind setting both `version` properties to `"7.0.0-beta.0"`?

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          contents += buildComponentCss(component);
+        }
+      }
+    }
+
+    const rules: Rule[] = [];
+
+    if (contents) {
+      contents = `${buildCommentBlock(
+        `TODO: The following componnent libraries introduced visual breaking changes in SKY UX 7. Each block of CSS reintroduces the styles that were changed or removed for backwards compatibility. You will need to do the following before migrating to the next major version of SKY UX:
```
Comment (Blackbaud-SteveBrush) [inline libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts]: `componnent`

File: libs/components/packages/src/schematics/migrations/migration-collection.json (hunk 1 above)
Comment (Blackbaud-PaulCrowder) [reply libs/components/packages/src/schematics/migrations/migration-collection.json]: Fixed.

File: libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts (hunk 2 above)
Comment (Blackbaud-PaulCrowder) [reply libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.ts]: Fixed.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**